jsapp/js/components/modalForms/translationTable.es6

```diff
@@ -49,14 +66,16 @@ export class TranslationTable extends React.Component {
     // add choice options to translation table
     if (choices && choices.length) {
       choices.forEach((choice) => {
+        let isLabelLocked = this.isLabelLocked(CHOICE_TYPE, choice.name);
```

Comment:
Instead of introducing this `CHOICE_TYPE` please make two separate functions `isChoiceLabelLocked` and `isRowLabelLocked` :)

Reply:
Okay so choices weren't actually being locked at all :sweat_smile:

This is because (and maybe we should change this) we don't apply the restriction to the choice lists, but rather to the rows that can potentially point to them, i.e., a select question can have the row restriction `choice_label_edit` but the `choice` in the code here doesn't know that.

I remedied it this way: https://github.com/kobotoolbox/kpi/pull/3209/commits/e685f5059372c520111cf64753d1482846375e8c -- I have it set that if a row has `choice_label_edit` locked then the choices in its linked choice list is also locked, even if another row does not have this restriction. Let me know what you think!

Reply:
Seems nice :)